js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -1253,15 +1253,29 @@
 
         // create graph section
         title = 'Create Graph';
-
+        var nameValidation = [
+          {
+            rule: Joi.string().regex(/^[a-zA-Z0-9_\-:\.@()\+,=;$!*\%'%]+$/),
```

Comment:
Third time this validation rule is used in the code.
It should really be placed into a function that is used from all 3 places.